confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/RegionGroup.java

```diff
@@ -28,27 +29,34 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
+import java.util.Map;
 import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.atomic.AtomicLong;
 
 public class RegionGroup {
 
   private final TRegionReplicaSet replicaSet;
 
+  // Map<TSeriesPartitionSlot, TTimePartitionSlot Count>
+  // For SchemaRegion, each SeriesSlot constitute a SchemaPartition.
   // For DataRegion, a SeriesSlot and a TimeSlot constitute a DataPartition.
   // Eg: A DataRegion contains SeriesSlot-1 which has TimeSlot-1, TimeSlot-2 and Timeslot-3,
   // then (SeriesSlot-1 -> TimeSlot-1) constitute a DataPartition.
-  // For SchemaRegion, each SeriesSlot constitute a SchemaPartition.
-  private final AtomicLong slotCount;
+  private final Map<TSeriesPartitionSlot, AtomicLong> slotCountMap;
```

Comment:
Maybe the value of Map<TSeriesPartitionSlot, AtomicLong> is not used?

Reply:
The value of Map<TSeriesPartitionSlot, AtomicLong> will be used to show the TimeSlotCount of show regions.